### PR TITLE
Feature/105 add tray icon

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Added tray icon.
Now the app runs in the background when you close all windows, unless you quit from the tray icon.

### CODE REVIEW

File: app/setup-tray-icon.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import path from 'path'
+import { app, Menu, Tray } from 'electron'
+
+let appIcon = null
+const iconPath = path.join(__dirname, '../docs/logo.png')
+
+function setupTrayIcon () {
```
Comment (koalalorenzo) [inline app/setup-tray-icon.js]: Disable here the trayicon on macOS! There is no need for that as the app on that platform will not quit if all the windows are closed.

File: app/setup-tray-icon.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import path from 'path'
+import { app, Menu, Tray } from 'electron'
+
+let appIcon = null
+const iconPath = path.join(__dirname, '../docs/logo.png')
+
```
Comment (koalalorenzo) [inline app/setup-tray-icon.js]: Docs? Maybe? :D

**Comment by koalalorenzo** [comment]
If I close the main window i can't see the trayicon on Ubuntu 16.04

screen shot 2018-05-08 at 19 49 54

on macOS (which should be disabled) I see a giant icon :D 

screen shot 2018-05-08 at 19 37 59

**Comment by 0x6431346e** [comment]
Let me know if the latest commit solves this :)

**Comment by koalalorenzo** [comment]
By installing it from the dpkg and then running it it works :)

**Pull request merged**